code/particle/util/ParticleProperties.cpp

```diff
@@ -62,6 +62,21 @@ void ParticleProperties::parse(bool nocreate) {
 			Warning(LOCATION, "Could not find curve '%s'", buf.c_str());
 		}
 	}
+
+	if (optional_string("+Rotation:")) {
+		SCP_string buf;
+		stuff_string(buf, F_NAME);
+		if (!stricmp(buf.c_str(), "DEFAULT")) {
+			m_rotation_type = RotationType::DEFAULT;
+		} else if (!stricmp(buf.c_str(), "RANDOM")) {
+			m_rotation_type = RotationType::RANDOM;
+		} else if (!stricmp(buf.c_str(), "SCREEN_ALIGNED")) {
```

Comment:
A little nitpicky, but i prefer omitting underscores from user facing strings where possible. "SCREEN ALIGNED"

Reply:
Okay.  How about all three, underscore, dash, and space?

Reply:
Works for me.